### PR TITLE
[API Deprecation] Deprecate candidates in convert module

### PR DESCRIPTION
**Pull request by peizhou001**
## Description
 Deprecate candidates in convert module

## Checklist
Please feel free to remove inapplicable items for your PR.
- [ ] The PR title starts with [$CATEGORY] (such as [NN], [Model], [Doc], [Feature]])
- [ ] I've leverage the [tools](https://docs.google.com/document/d/1iHyj7zlmygKSk5gBPsqIqL5ASPzJSPREaNT_QdsiYA4/edit) to beautify the python and c++ code.
- [ ] The PR is complete and small, read the [Google eng practice (CL equals to PR)](https://google.github.io/eng-practices/review/developer/small-cls.html) to understand more about small PR. In DGL, we consider PRs with less than 200 lines of core code change are small (example, test and documentation could be exempted).
- [ ] All changes have test coverage
- [ ] Code is well-documented
- [ ] To the best of my knowledge, examples are either not affected by this change, or have been fixed to be compatible with this change
- [ ] Related issue is referred in this PR
- [ ] If the PR is for a new model/paper, I've updated the example index [here](../examples/README.md).

### CODE REVIEW

File: python/dgl/convert.py [hunk 1]
```diff
@@ -157,13 +147,6 @@ def graph(data,
     from_networkx
     """
     # Deprecated arguments
```
Comment (mufeili) [inline python/dgl/convert.py:149]: Remove the comment

**Comment by mufeili** [comment]
Have we passed the example tests?

File: python/dgl/convert.py (hunk 1 above)
Comment (peizhou001) [reply python/dgl/convert.py:149]: fixed

File: python/dgl/nn/mxnet/conv/gmmconv.py [hunk 2]
```diff
@@ -93,7 +93,7 @@ class GMMConv(nn.Block):
     >>> # Case 2: Unidirectional bipartite graph
     >>> u = [0, 1, 0, 0, 1]
     >>> v = [0, 1, 2, 3, 2]
-    >>> g = dgl.bipartite((u, v))
+    >>> g = g = dgl.heterograph({('_N', '_E', '_N'):(u, v)})
```
Comment (mufeili) [inline python/dgl/nn/mxnet/conv/gmmconv.py:96]:
```suggestion
    >>> g = dgl.heterograph({('_N', '_E', '_N'):(u, v)})
```

File: python/dgl/nn/mxnet/conv/graphconv.py [hunk 3]
```diff
@@ -120,7 +120,7 @@ class GraphConv(gluon.Block):
     >>> # Case 2: Unidirectional bipartite graph
     >>> u = [0, 1, 0, 0, 1]
     >>> v = [0, 1, 2, 3, 2]
-    >>> g = dgl.bipartite((u, v))
+    >>> g = g = dgl.heterograph({('_N', '_E', '_N'):(u, v)})
```
Comment (mufeili) [inline python/dgl/nn/mxnet/conv/graphconv.py:123]: Ditto

File: python/dgl/nn/mxnet/conv/nnconv.py [hunk 4]
```diff
@@ -75,7 +75,7 @@ class NNConv(nn.Block):
     >>> # Case 2: Unidirectional bipartite graph
     >>> u = [0, 1, 0, 0, 1]
     >>> v = [0, 1, 2, 3, 2]
-    >>> g = dgl.bipartite((u, v))
+    >>> g = g = dgl.heterograph({('_N', '_E', '_N'):(u, v)})
```
Comment (mufeili) [inline python/dgl/nn/mxnet/conv/nnconv.py:78]: Ditto

File: python/dgl/nn/mxnet/conv/sageconv.py [hunk 5]
```diff
@@ -76,7 +76,7 @@ class SAGEConv(nn.Block):
     >>> # Case 2: Unidirectional bipartite graph
     >>> u = [0, 1, 0, 0, 1]
     >>> v = [0, 1, 2, 3, 2]
-    >>> g = dgl.bipartite((u, v))
+    >>> g = g = dgl.heterograph({('_N', '_E', '_N'):(u, v)})
```
Comment (mufeili) [inline python/dgl/nn/mxnet/conv/sageconv.py:79]: Ditto

File: python/dgl/nn/pytorch/conv/dotgatconv.py [hunk 6]
```diff
@@ -99,7 +99,7 @@ class DotGatConv(nn.Module):
     >>> # Case 2: Unidirectional bipartite graph
     >>> u = [0, 1, 0, 0, 1]
     >>> v = [0, 1, 2, 3, 2]
-    >>> g = dgl.bipartite((u, v))
+    >>> g = g = dgl.heterograph({('_N', '_E', '_N'):(u, v)})
```
Comment (mufeili) [inline python/dgl/nn/pytorch/conv/dotgatconv.py:102]: Ditto

File: python/dgl/nn/pytorch/conv/edgeconv.py [hunk 7]
```diff
@@ -81,7 +81,7 @@ class EdgeConv(nn.Module):
     >>> # Case 2: Unidirectional bipartite graph
     >>> u = [0, 1, 0, 0, 1]
     >>> v = [0, 1, 2, 3, 2]
-    >>> g = dgl.bipartite((u, v))
+    >>> g = g = dgl.heterograph({('_N', '_E', '_N'):(u, v)})
```
Comment (mufeili) [inline python/dgl/nn/pytorch/conv/edgeconv.py:84]: Ditto

File: python/dgl/nn/pytorch/conv/nnconv.py [hunk 8]
```diff
@@ -72,7 +72,7 @@ class NNConv(nn.Module):
     >>> # Case 2: Unidirectional bipartite graph
     >>> u = [0, 1, 0, 0, 1]
     >>> v = [0, 1, 2, 3, 2]
-    >>> g = dgl.bipartite((u, v))
+    >>> g = g = dgl.heterograph({('_N', '_E', '_N'):(u, v)})
```
Comment (mufeili) [inline python/dgl/nn/pytorch/conv/nnconv.py:75]: Ditto

File: python/dgl/nn/pytorch/conv/sageconv.py [hunk 9]
```diff
@@ -83,7 +83,7 @@ class SAGEConv(nn.Module):
     >>> # Case 2: Unidirectional bipartite graph
     >>> u = [0, 1, 0, 0, 1]
     >>> v = [0, 1, 2, 3, 2]
-    >>> g = dgl.bipartite((u, v))
+    >>> g = g = dgl.heterograph({('_N', '_E', '_N'):(u, v)})
```
Comment (mufeili) [inline python/dgl/nn/pytorch/conv/sageconv.py:86]: Ditto

File: python/dgl/nn/mxnet/conv/gmmconv.py (hunk 2 above)
Comment (peizhou001) [reply python/dgl/nn/mxnet/conv/gmmconv.py:96]: fixed

File: python/dgl/nn/mxnet/conv/graphconv.py (hunk 3 above)
Comment (peizhou001) [reply python/dgl/nn/mxnet/conv/graphconv.py:123]: fixed

File: python/dgl/nn/mxnet/conv/nnconv.py (hunk 4 above)
Comment (peizhou001) [reply python/dgl/nn/mxnet/conv/nnconv.py:78]: fixed

File: python/dgl/nn/mxnet/conv/sageconv.py (hunk 5 above)
Comment (peizhou001) [reply python/dgl/nn/mxnet/conv/sageconv.py:79]: fixed

File: python/dgl/nn/pytorch/conv/dotgatconv.py (hunk 6 above)
Comment (peizhou001) [reply python/dgl/nn/pytorch/conv/dotgatconv.py:102]: fixed

File: python/dgl/nn/pytorch/conv/edgeconv.py (hunk 7 above)
Comment (peizhou001) [reply python/dgl/nn/pytorch/conv/edgeconv.py:84]: fixed

File: python/dgl/nn/pytorch/conv/gmmconv.py [hunk 10]
```diff
@@ -91,7 +91,7 @@ class GMMConv(nn.Module):
     >>> # Case 2: Unidirectional bipartite graph
     >>> u = [0, 1, 0, 0, 1]
     >>> v = [0, 1, 2, 3, 2]
-    >>> g = dgl.bipartite((u, v))
+    >>> g = g = dgl.heterograph({('_N', '_E', '_N'):(u, v)})
```
[the remark this answers is not in the document]
Comment (peizhou001) [reply python/dgl/nn/pytorch/conv/gmmconv.py:94]: fixed

File: python/dgl/nn/pytorch/conv/nnconv.py (hunk 8 above)
Comment (peizhou001) [reply python/dgl/nn/pytorch/conv/nnconv.py:75]: fixed

File: python/dgl/nn/pytorch/conv/sageconv.py (hunk 9 above)
Comment (peizhou001) [reply python/dgl/nn/pytorch/conv/sageconv.py:86]: fixed

File: python/dgl/nn/tensorflow/conv/graphconv.py [hunk 11]
```diff
@@ -122,7 +122,7 @@ class GraphConv(layers.Layer):
     >>> u = [0, 1, 0, 0, 1]
     >>> v = [0, 1, 2, 3, 2]
     >>> with tf.device("CPU:0"):
-    ...     g = dgl.bipartite((u, v))
+    ...     g = g = dgl.heterograph({('_N', '_E', '_N'):(u, v)})
```
[the remark this answers is not in the document]
Comment (peizhou001) [reply python/dgl/nn/tensorflow/conv/graphconv.py:125]: fixed

File: python/dgl/nn/tensorflow/conv/sageconv.py [hunk 12]
```diff
@@ -76,7 +76,7 @@ class SAGEConv(layers.Layer):
     >>> with tf.device("CPU:0"):
     >>>     u = [0, 1, 0, 0, 1]
     >>>     v = [0, 1, 2, 3, 2]
-    >>>     g = dgl.bipartite((u, v))
+    >>>     g = g = dgl.heterograph({('_N', '_E', '_N'):(u, v)})
```
[the remark this answers is not in the document]
Comment (peizhou001) [reply python/dgl/nn/tensorflow/conv/sageconv.py:79]: fixed

**Review by mufeili — APPROVED** [review]

**Pull request merged**